src/deepsparse/yolo/README.md

```diff
@@ -0,0 +1,179 @@
+
+This integration currently supports the original YOLOv5 the updated V6.1 architectures
+
+## Getting Started
+
+
+Before you start your adventure with the DeepSparse Engine, make sure that your machine is 
+compatible with our [hardware requirements](https://docs.neuralmagic.com/deepsparse/source/hardware.html).
+
+### Installation
+
+```pip install deepsparse```
+
+### Model Format
+By default, to deploy YOLOv5 using DeepSparse Engine it is required to supply the model in the ONNX format. 
+This grants the Engine the flexibility to serve any model in a framework-agnostic environment. 
+
+Below we describe two possibilities to obtain the required ONNX model.
+
+### Exporting the onnx file from the contents of a local directory
+This pathway is relevant if you intend to deploy a model created using [SparseML](https://github.com/neuralmagic/sparseml) library. 
+For more information refer to the appropriate YOLOv5 integration documentation in SparseML.
+1. After training your model with `SparseML`, locate the `.pt` file for the model you'd like to export
+2. Run the `SparseML` integrated YOLOv5 onnx export script the CLI command as below
+```bash
+sparseml.yolov5.export_onnx --weights path/to/your/model --dynamic
+```
+This will create an `.onnx` file in the same directory and with the same root name as the 
+model weights file. 
+
+###  Directly using the SparseZoo stub
+Alternatively, you can skip the onnx model export process by downloading all the required model data directly from Neural Magic's [SparseZoo](https://sparsezoo.neuralmagic.com/).
+Example:
+```python
+from sparsezoo import Zoo
+
+# you can lookup an appropriate model stub here: https://sparsezoo.neuralmagic.com/
+model_stub = "zoo:cv/detection/yolov5-l/pytorch/ultralytics/coco/pruned-aggressive_98"
+# directly download the model data to your local directory
+model = Zoo.download_model_from_stub(model_stub)
+
+# the onnx model file is there, ready for deployment
+import os 
+os.path.isfile(os.path.join(model.dir_path, "model.onnx"))
+>>True
+```
+
+
+## Deployment
+
+### Python API
+Python API is the default interface for running the inference with the DeepSparse Engine. We can use it to run inference on local images. If you don't have an image ready, pull a sample image down with
+
+```
+wget -O abbey_road.jpg  https://upload.wikimedia.org/wikipedia/en/4/42/Beatles_-_Abbey_Road.jpg
+```
+
+[List of the YOLOv5 SparseZoo Models](
+https://sparsezoo.neuralmagic.com/?page=1&domain=nlp&sub_domain=question_answering)
+
+```python
+from deepsparse.pipeline import Pipeline
+
+model_stub = "zoo:cv/detection/yolov5-l/pytorch/ultralytics/coco/pruned-aggressive_98"
+images = ["abbey_road.jpg"]
+
+od_pipeline = Pipeline.create(
+    task="yolo",
+    model_path=model_stub,
+)
+
+pipeline_outputs = od_pipeline(images=images, iou_thres=0.6, conf_thres=0.001)
+print(pipeline_outputs)
+```
+
+### Annotate CLI
```

Comment:
would be nice to flesh this section out a little more. maybe a picture of the annotation script in action